client/src/components/Explorer/Explorer.scss

```diff
@@ -1,5 +1,3 @@
-// scss-lint:disable DeclarationOrder
```

Comment:
I decided to leave this one out for now, as the configuration mapping from the existing setting is unclear. Probably best to add it in its own PR.